segmentation_labeling_app/rois/rois.py

```diff
@@ -142,27 +142,27 @@ def create_dilated_contour_mask(self, stroke_size: int,
     def get_edge_points(self, threshold: float,
```

Comment:
this function is now _almost_ just a wrapper of `findContours` which isn't terrible, but, since it is only called once, you might save some code and tests just by getting rid of it and including your logic in the calling function.